package com.exception;


class Car{
	int num =1234;
}
public class ExceptionTest {
	public static void main(String[] args) {
		int result = 0;
		int a =13, b=0;
		Car c = null;
		
		try {
			result = a/b;
			System.out.println(c.num);
		}catch(ArithmeticException e) {
			result = 9999;
		}catch(NullPointerException e) {
			c = new Car();
		}
		System.out.println(c.num);
		System.out.println(result);
		
	}
}
